src/cascadia/TerminalApp/PaletteItem.h

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+#include "inc/cppwinrt_utils.h"
+#include "PaletteItem.g.h"
+
+namespace winrt::TerminalApp::implementation
+{
+    struct PaletteItem : PaletteItemT<PaletteItem>
+    {
+    public:
+        WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
+
+        OBSERVABLE_GETSET_PROPERTY(winrt::hstring, Name, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(winrt::hstring, Icon, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(winrt::hstring, KeyChordText, _PropertyChangedHandlers);
+        OBSERVABLE_GETSET_PROPERTY(winrt::hstring, FilterableValue, _PropertyChangedHandlers);

```

Comment:
FilterableValue not mentioned in the idl!

Reply:
Sorry.. It was a part of the idl at some moment.. but then I decided to postpone this feature (until we generalize the filterable list.. and realized it will be implemented differently).. unfortunately, I forgot to cleanup it from the header..